anvil/config.py

```diff
@@ -241,13 +175,16 @@ def parse_sample_interval(self, interval: (int, type(None))):
         log.warning(f"Using user specified sample_interval: {interval}")
         return interval
 
-    def parse_n_boot(self, n_boot: int):
+    def parse_bootstrap_sample_size(self, n_boot: int):
         """Size of the bootstrap sample."""
         if n_boot < 2:
-            raise ConfigError("n_boot must be greater than 1")
-        log.warning(f"Using user specified n_boot: {n_boot}")
+            raise ConfigError("bootstrap sample size must be greater than 1")
+        log.warning(f"Using user specified bootstrap sample size: {n_boot}")
         return n_boot
 
+    def produce_bootstrap_seed(self):
+        return randint(0, maxsize)
```

Comment:
I would prefer if there was an option to seed the seed in case you really wanted to get the same results. Having said that we can leave that as a desirable in the future.

Reply:
yeah good point. How about just "if seed not provided in runcard: return randint"?

Reply:
Good idea, I'll push something